README.md

```diff
@@ -8,7 +8,7 @@ Look at the Merge Confidence badges before merging to:
 Merge Confidence finds and flags undeclared breaking releases.
 It analyzes test and release adoption data across WhiteSource Renovate’s early-adopting user base.
 
-![Renovate PR with Merge Confidence badges](https://github.com/whitesource/merge-confidence/blob/main/assets/merge-confidence.png)
+![Renovate PR with Merge Confidence badges](https://github.com/whitesource/merge-confidence/raw/main/assets/merge-confidence.png)
```

Comment:
```suggestion
![Renovate PR with Merge Confidence badges](https://raw.githubusercontent.com/whitesource/merge-confidence/main/assets/merge-confidence.png)
```

@rarkins Maybe remove the redirect is even better?

Reply:
Does GitHub allow third party sites to serve images?

Reply:
yes, at least for `raw.githubusercontent.com`